### PR TITLE
Updating Developer Setup Guide to use Pulplift

### PR DESCRIPTION
**Pull request by CodeHeeler**
Instructs the developer to setup their environment with Pulplift

re #4234
https://pulp.plan.io/issues/4234

### CODE REVIEW

File: docs/contributing/dev-setup/index.rst [hunk 1]
```diff
@@ -3,20 +3,42 @@
...
+Clone the 'pulplift', 'pulp', and 'pulp-file' (or any plugins that you'll be working on) repos into
+the same directory so that they are peers.
 
-`Vagrant <https://docs.vagrantup.com/>`_ is used to create and manage development virtual machines.
-Configuring Vagrant is beyond the scope of this document, but you can find suggestions in the
-comments of ``Vagrantfile.example``  in the aforementioned `pulp/devel
-repository <https://github.com/pulp/devel/>`_ and on the `wiki
-<https://pulp.plan.io/projects/pulp/wiki/Developer_Install_Options>`_.
+Navigate into the pulplift directory. Run the setup.sh script which will clone the 'forklift' and
+'ansible-pulp3' repos::
```
Comment (dkliban) [inline docs/contributing/dev-setup/index.rst]: Please add a link to ansible-pulp3 repo

File: docs/contributing/dev-setup/index.rst [hunk 2]
```diff
@@ -3,20 +3,42 @@
...
+
+    ./setup.sh
+
+The parent directory of 'pulplift' is going to be mounted into the vagrant box at
+/home/vagrant/devel. Any plugins that need to be installed should be added to the
+forklift/ansible-pulp3/source-install.yml playbook. The paths to the source of the plugins should
```
Comment (dkliban) [inline docs/contributing/dev-setup/index.rst]: s/forklift/pulplift/

File: docs/contributing/dev-setup/index.rst [hunk 3]
```diff
@@ -3,20 +3,42 @@
 Developer Setup
 ===============
 
-To ease developer setup, we have Ansible roles, playbooks and a Vagrantfile in our `pulp/devel
-<https://github.com/pulp/devel/>`_ repository. Use the :ref:`quickstart` to dive in with an example
-configuration.
+To ease developer setup, we have `Pulplift <https://github.com/ehelms/pulplift>`_ which is based on
```
Comment (bmbouter) [inline docs/contributing/dev-setup/index.rst]: Does it make any sense to move this into the Pulp or and update this link to that pre-merge?

Comment (dkliban) [reply docs/contributing/dev-setup/index.rst]: Let's update the link after the repo is transferred.

**Pull request merged**